### DISCUSSION TITLE
only getting some stock

### DISCUSSION THREAD

**Discussion by kerchew** [jef/streetmerchant#1266] [Help] [answered]
Having the issue of only checking nvidia stock for 3070,3080,3090 and nothing else.

Anyone else having that issue?
how would I solve that?

**Comment by kerchew** [comment, marked as answer]
https://jef.codes/streetmerchant/reference/filter/

Nvm the .env file needed editing. thought it had preset all. Works great.

**Reply by jef** [reply]
Awesome! Enjoy 😃
